x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/home/hooks/use_available_packages.tsx

```diff
@@ -146,10 +150,29 @@ export const useAvailablePackages = ({
     });
   }
 
-  const eprIntegrationList = useMemo(
-    () => packageListToIntegrationsList(eprPackages?.items || []),
-    [eprPackages]
-  );
+  const eprIntegrationList = useMemo(() => {
+    // Filter out agentless only packages and policy templates if agentless is not available
+    const packages = isAgentlessEnabled
+      ? eprPackages?.items
+      : eprPackages?.items
```

Comment:
nit: I think the filtering logic could be extracted to a separate function, to separate better the agentless related logic

Reply:
moved this logic out